src/style/base/_base.scss

```diff
@@ -48,25 +58,33 @@ h1, h2, h3, h4, h5, h6 {
 }
 
 h1 {
-    font-size: 60px;
+    font-size: 110px;
     line-height: 72px;
 }
 
 h2 {
-    font-size: 34px;
+    font-size: 110px;
     line-height: 44px;
 }
 
-p, a {
+
+p {
     font-family: $body-font;
 }
 
 a {
+    font-family: $header-font;
+
     &:hover {
         color: inherit;
     }
 }
 
+.ital {
+    font-family: $header-font2;
+    font-size: 103px;
```

Comment:
Depending on the change above for the headers, might have to add `font-weight: 500` if a bolded font-weight is added in h1 or h2 since this class is applied on elements within the header tags. But regardless, it doesn't hurt to be specific?